src/textual/app.py

```diff
@@ -1156,7 +1156,10 @@ def _print(self, text: str, stderr: bool = False) -> None:
             stderr: True if the print was to stderr, or False for stdout.
         """
         if self._devtools_redirector is not None:
-            self._devtools_redirector.write(text)
+            current_frame = inspect.currentframe()
+            self._devtools_redirector.write(
+                text, current_frame.f_back if current_frame is not None else None
```

Comment:
So, the previous frame is the one where `print` was called?
Can you briefly explain how a call to `print` ends up here?

Reply:
> Can you briefly explain how a call to print ends up here?

Not off the top of my head; I don't recall going near that.